package.json

```diff
@@ -18,6 +19,9 @@
     "react-icons": "^4.3.1",
     "react-router-dom": "^6.2.2",
     "react-scripts": "^4.0.3",
+    "react-slick": "^0.28.1",
+    "slick": "^1.12.2",
```

Comment:
You might not need to explicitly use this(`slick`) library as `react-slick` will install this for you.